region/wreckedship/main/Wrecked Ship Energy Tank Room.json

```diff
@@ -687,6 +740,72 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 2],
+      "name": "Shinespark to Item (X-Mode)",
+      "requires": [
+        {"or": [
+          "Gravity",
+        {"and": [
+          {"not": "f_DefeatedPhantoon"},
+          "canRiskPermanentLossOfAccess"
+        ]}
+        ]},
```

Comment:
In this scenairo: if phantoon is defeated, you can do this for free, and if phantoon is not defeated, you don't need gravity. So this chunk shouldnt be necessary. This wouldn't be the case if you are trying to carry a blue suit, but we havent made a pass of that yet.
```suggestion
```